logging/bin/deploy_logging.sh

```diff
@@ -31,8 +31,10 @@ checkDefaultStorageClass
 # Create namespace if it doesn't exist
 if [ "$(kubectl get ns $LOG_NS -o name 2>/dev/null)" == "" ]; then
   kubectl create ns $LOG_NS
-fi
 
+  #Container Security: Disable serviceAccount Token Automounting
+  disable_sa_token_automount $LOG_NS default
```

Comment:
Currently, only doing it if we're creating the namespace...should we do it always?  Would "patch" existing deployments if doing an UIP.  If user didn't want that to happen, they could set SEC_DISABLE_SA_TOKEN_AUTOMOUNT to "false".  Or, they could manually patch things after the fact.  Thoughts?

Reply:
I would imagine that this would be fine to create if we're creating the namespace.  If the customer's doing an UIP, I could see them wanting to preserve the previous one (unless that could become troublesome).

What's our stance on customer's deploying cluster logging or monitoring in an existing namespace?  Do we frown upon it or is it deemed okay?

If we tell our customers that it's okay to deploy logging in an existing namespace, then this behavior could be problematic.  If not, then this concern can be disregarded.

Reply:
Users are free to deploy Viya Monitoring into any namespace they want.  That said, I think we'd prefer/recommend they keep the Viya Monitoring stuff separated from other applications (e.g. don't deploy Viya Monitoring within a namespace that is being used for SAS Viya).  Some users deploy all of Viya Monitoring (both metric and log monitoring components) into a single namespace.  That shouldn't be a problem either. 

As for UIP scenarios, the only difference between those and a "clean" deploy is that this update will "skip" setting the automount property for the default serviceAccount in that case.  But that won't cause any problems.  We _could_ move the line of code that does that to outside of the IF-block that currently only does it when creating the namespace.  I suspect most UIP scenarios are updating a deployment where we originally created the namespace as opposed to users having done that.  So, in those cases, setting the automount property would probably be just fine.  But just in case the user created the namespace and may have special needs, it seemed better to skip setting that property in those scenarios.